pandas/core/indexes/accessors.py

```diff
@@ -102,7 +102,9 @@ def _delegate_property_get(self, name):
         else:
             index = self._parent.index
         # return the result as a Series, which is by definition a copy
-        result = Series(result, index=index, name=self.name).__finalize__(self._parent)
+        result = Series(result, index=index, name=self.name, copy=False).__finalize__(
```

Comment:
The comment just above should be updated then. But also wondering if we have properties that can be a view, in which case we shouldn't necessarily do `copy=False`? (but rather attach a ref to `self._parent`)

Reply:
Looking through the datetimelike properties/methods, I think theoretically things like `round`/`ceil`/`floor` _could_ return a view (if nothing needs to be rounded), but that's not an optimization we currently have I think. 
Also `tz_convert`/`tz_localize` could theoretically return a view (but in practice also do not do that, from a quick test)

```
In [41]: s = pd.Series(pd.date_range("2012-01-01", periods=3, tz="UTC"))

In [43]: s._values._ndarray
Out[43]: 
array(['2012-01-01T00:00:00.000000000', '2012-01-02T00:00:00.000000000',
       '2012-01-03T00:00:00.000000000'], dtype='datetime64[ns]')

In [44]: s2 = s.dt.tz_convert("Europe/Brussels")

In [45]: np.shares_memory(s._values._ndarray, s2._values._ndarray)
Out[45]: False
```

Reply:
Actually they return views if it's a no-op. So will remove the copy=False for now